### PR TITLE
Included correct image url in GetCohortById and GetAllCohorts

### LINKED ISSUE #60: GetAllCohort bug - cohort.user.profileImage is shown as null even though it contains string

### PR DESCRIPTION
**Pull request by Muzea001**

### CODE REVIEW

**Review by JHalvor — CHANGES_REQUESTED** [review]
will have to wait, replace seed generation for image url to a .csv file with hard coded image url strings

File: prisma/seed.js
Comment (JHalvor) [inline prisma/seed.js:1]: dont need this, put in other issue

File: src/domain/cohort.js
Comment (JHalvor) [inline src/domain/cohort.js:1]: good! keep!

**Review by JHalvor — CHANGES_REQUESTED** [review]
remove the seed change and put it in other issue for seeding.
other file is good!

**Review by JHalvor — APPROVED** [review]
Looks good tested and works

**Review by Brandsegg — APPROVED** [review]
Very nice!

**Pull request merged**